src/server.prod.js

```diff
@@ -45,9 +47,17 @@ class ProdServer {
     const nockPath = this.config.nockPath;
     const accessLogs = this.config.accessLogs || 'clf';
     const renderFn = this.getRenderFunction();
+
     if (!this.config.isCDN) {
       this.setupStaticFiles(app);
     };
+
+    if(this.config.compressHTML) {
```

Comment:
```suggestion
    if (this.config.compressHTML) {
```

Sorry for nitpicking